app/views/layouts/_sidebar.html.erb

```diff
@@ -23,6 +23,7 @@
         <ul>
           <%= render(Sidebar::LinkComponent.new(title: "Supervisors", icon: "network", path: supervisors_path, render_check: policy(Supervisor).index?)) %>
           <%= render(Sidebar::LinkComponent.new(title: "Volunteers", icon: "heart-filled", path: volunteers_path, render_check: policy(Volunteer).index?)) %>
+           <%= render(Sidebar::LinkComponent.new(title: "Learning Hours", icon: "timer", path: learning_hours_path, render_check: current_user.casa_admin? || current_user.supervisor?)) %>
```

Comment:
Can you use the policy here so we don’t have permissions in two different places?

```
policy(LearningHour).index?
```

Also, slight formatting change request to remove one whitespace and have this line of code line up with the one above.

Reply:
Have removed this code and updated the Learning Hour link already defined below in code for volunteer with policy. If I do this one it will show as top link in Volunteer menu which might not be correct.

Reply:
Have updated screenshot in main description to represent current order of Learning Hours menu for Supervisor and Casa Admin as we are using same link in sidebar using policy.